go-controller/pkg/ovn/gress_policy.go

```diff
@@ -0,0 +1,193 @@
+package ovn
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
+	v1 "k8s.io/api/core/v1"
+	knet "k8s.io/api/networking/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+type gressPolicy struct {
+	namespace  string
+	name       string
+	policyType knet.PolicyType
+	idx        int
+
+	// peerAddressSet points to the addressSet that holds all peer pod
+	// IP addresess.
+	peerAddressSet *addressSet
+
+	// nsAddressSets holds the names of all namespace address sets
+	nsAddressSets sets.String
+
+	// sortedPeerAddressSets has the sorted peerAddressSets
+	sortedPeerAddressSets []string
+
+	// portPolicies represents all the ports to which traffic is allowed for
+	// the rule in question.
+	portPolicies []*portPolicy
```

Comment:
port is a little ambiguous, and this isn't really a policy. It's holding an L4 port so how about "L4Ports" or something like that?

Reply:
@trozet this is simply a move of code from policy.go to gress_policy.go so I'd prefer to leave it as-is for now. Could be cleaned up in a follow-up though.

Reply:
> this is simply a move of code from policy.go to gress_policy.go so I'd prefer to leave it as-is for now

It's not *just* a move though. eg, you changed the arguments to `newGressPolicy`. It would be easier to review if the just-move was separate

Reply:
I mean it's one right click refactor in goland :) But I can change it later if you don't want to touch it now.

Reply:
Yeah I'd rather not.